### DISCUSSION TITLE
Too many first aid kits for a contract - Paradox75831004

### DISCUSSION THREAD

**Discussion by Malcolm-Q** [Malcolm-Q/LC-LateGameUpgrades#221] [Ideas]
There are too many first aid kits, can you somehow reduce them for the use contract? So that there is one with 3 charges for three survivors, for example. But nito in the store is expensive, you can farm it hard.

**Comment by WhiteSpike** [comment]
Closing due to 3.3.0 release implementing option to configure how many medkits it can spawn during an extraction contract. This way, you can decide to not have any medkits spawned and be forced to buy one for the contract or have to find only one.
